config/config.go

```diff
@@ -2,20 +2,23 @@
 package config
 
 import (
-	"sync"
-
+	"errors"
+	"github.com/companieshouse/chs.go/log"
 	"github.com/companieshouse/gofigure"
+	"sync"
 )
 
 var cfg *Config
 var mtx sync.Mutex
+var mandatoryElementMissing bool
 
 // Config defines the configuration options for this service.
 type Config struct {
-	BindAddr          string   `env:"BIND_ADDR" flag:"bind-addr" flagDesc:"Bind address"`
-	BrokerAddr        []string `env:"KAFKA_BROKER_ADDR" flag:"broker-addr" flagDesc:"Kafka broker address"`
-	SchemaRegistryURL string   `env:"SCHEMA_REGISTRY_URL" flag:"schema-registry-url" flagDesc:"URL for Schema Registry"`
-	OfficerDeltaTopic string `env:"OFFICER_DELTA_TOPIC" flag:"officer-delta-topic" flagDesc:"Topic for the officer delta"`
+	BindAddr          string    `env:"BIND_ADDR" flag:"bind-addr" flagDesc:"Bind address"`
+	BrokerAddr        []string  `env:"KAFKA_BROKER_ADDR" flag:"broker-addr" flagDesc:"Kafka broker address (Comma separated list if there are more than one addresses)"`
```

Comment:
Kafka broker address (Comma separated list if there are more than one addresses
>
Kafka broker address (Comma separated list if there is more than one address